README.md

```diff
@@ -18,6 +18,21 @@ The current v1 release is `1.0.0-rc2` on pub. The last stable version so far is
 
 A minimalistic Flutter game engine.
 
+## Documentation
+
+Note that the documentation in the master branch of this repo is newer than the latest released version.
+
+Here you can find the documentation for different versions:
+- Latest stable version (0.28.x): [Flame-engine website](https://flame-engine.org/)
+- Latest [0.28.x release](https://github.com/flame-engine/flame/tree/master-v0.x/doc) GitHub docs
```

Comment:
This could be a hassle to maintain as we release more version, I think we could just omit the versions, what do you think?

Reply:
Good point, removing them!